app/models/user.rb

```diff
@@ -215,6 +211,10 @@ def self.search(search)
     (q_user_names + q_team_names).uniq
   end
 
+  def student_team
+    self.teams.last if self.student?
```

Comment:
I'm not sure if this can guarantee to always select the "right" team... as far as I remember, `team` (hopefully) does not have a default scope.

What would be the team you want to retrieve here? The "current" one as in the one active in this current season? Why not use
```ruby
teams.in_current_season.last if student?
```